### PR TITLE
Overview subsection of the explanation section

### PR DESCRIPTION
**Pull request by nibble0101**
# Improve the Introduction and Explanation sections project

| Section | Sub-section | Technical Writer |
| --- | --- | --- |
|Explanations | Overview of the explanation section |  Chris Estepa |

This  is **Overview** sub-section of the **Explanations** section

image
image
image

### CODE REVIEW

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Great work @nibble0101 and @chris-4444 : )

**Review by proudofsimin — APPROVED** [review]
Thank you!

**Comment by lijiarui** [comment]
hi @Rohitesh-Kumar-Jain As GSoD Project PR Review Rules in https://github.com/wechaty/wechaty.js.org/issues/1045

This PR should follow our rules and get enough approval, then I will review this PR. So I remove the `ready to merge` tag and feel free to add this tag when it gets enough approval.

**Comment by Rohitesh-Kumar-Jain** [comment]
@chris-4444 I would like you to once go through the PR, and approve when you think it's ready to get merged.

**Review by chris-4444 — APPROVED** [review]
LGTM.

File: docusaurus/docs/explainations/overview.mdx [hunk 1]
```diff
@@ -1,7 +1,29 @@
 ---
 slug: /explainations/
-title: Wechaty explainations overview
-sidebar_label: 'Explainations: Overview'
+title: Overview of Explanation section
+sidebar_label: 'Overview of Explanation section'
 ---
 
-Explanations will discuss key topics and concepts at a fairly high level and provide useful background information and explanation.
+![Wechaty: Conversational RPA SDK for Chatbot Makers](/img/wechaty-logo.svg)
```
Comment (lijiarui) [inline docusaurus/docs/explainations/overview.mdx]: I feel like the wechaty logo appears too much in the doc, how about we change a more related image to the explanation part if you think we need add image here.

File: docusaurus/docs/explainations/overview.mdx [hunk 2]
```diff
@@ -1,7 +1,29 @@
...
+
+The **Explanation section** discusses topics that broaden your knowledge of Wechaty. Non-technical facts and some tidbits of useful information are found here to give you more clarity and understanding of what Wechaty is all about.
+
+## [Life cycle](#placeholder-link)
+
```
Comment (lijiarui) [inline docusaurus/docs/explainations/overview.mdx]: I'm not sure if we need to add a life cycle here. 

If you are saying wechaty lifecycle, it is necessary, but if you are saying chatbot lifecycle, it is not that related to wechaty doc.

**Review by lijiarui — CHANGES_REQUESTED** [review]
It seems explanation overview is talking about chatbot not Wechaty.

I think we should add more Wechaty part instead of chatbot part.

File: docusaurus/docs/explainations/overview.mdx (hunk 1 above)
Comment (nibble0101) [reply docusaurus/docs/explainations/overview.mdx]: Removed logo

File: docusaurus/docs/explainations/overview.mdx (hunk 2 above)
Comment (nibble0101) [reply docusaurus/docs/explainations/overview.mdx]: Can you clarify on this? Should we remove it from the explanation section completely?

**Comment by chris-4444** [comment]
@lijiarui  May I know what do you mean by adding more about the Wechaty part? Is it pertaining to Wechaty as an organization itself? If yes, it's already covered in [**What is Wechaty**](https://github.com/wechaty/wechaty.js.org/pull/871) and [**What can you do with Wechaty**.](https://github.com/wechaty/wechaty.js.org/pull/873) In any case that the information we've provided is lacking, would it be okay if you explain it to us in more detail? Kindly advise so that we can revise. Thanks a lot!  😉

**Comment by lijiarui** [comment]
@chris-4444 

> May I know what do you mean by adding more about the Wechaty part

May I know where is this comment related to this PR?

**Comment by chris-4444** [comment]
@lijiarui Yep, I'm referring to your comment above re PR 915.  We already have covered some things about Wechaty (e.g., [**What is Wechaty**](https://github.com/wechaty/wechaty.js.org/pull/871) and [**What can you do with Wechaty**.](https://github.com/wechaty/wechaty.js.org/pull/873).   The ones reflected in the Explanation overview are based on the topics reflected in Explanations before as well. In case we miss something, please let us know.  😉

image


> @chris-4444
> 
> > May I know what do you mean by adding more about the Wechaty part
> 
> May I know where is this comment related to this PR?



> @lijiarui May I know what do you mean by adding more about the Wechaty part? Is it pertaining to Wechaty as an organization itself? If yes, it's already covered in [**What is Wechaty**](https://github.com/wechaty/wechaty.js.org/pull/871) and [**What can you do with Wechaty**](https://github.com/wechaty/wechaty.js.org/pull/873) In any case that the information we've provided is lacking, would it be okay if you explain it to us in more detail? Kindly advise so that we can revise. Thanks a lot! 😉

File: docusaurus/docs/explainations/overview.mdx (hunk 2 above)
Comment (lijiarui) [reply docusaurus/docs/explainations/overview.mdx]: If we cannot explain this, I think we should remove this.

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
Hi, @chris-4444  I feel a little bit confused about you asking me what should add for this part.

I think if we don't have more to talk, then we should not add the explanation here.

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Review by chris-4444 — APPROVED** [review]

**Comment by chris-4444** [comment]
@lijiarui @huan @Rohitesh-Kumar-Jain kindly merge because merging is disabled for me and @nibble0101. thanks.

**Comment by huan** [comment]
@chris-4444 The reason that you can not merge this PR is that @lijiarui has a "**request changes**" on it.

Please contact @lijiarui and get the approval from her, then this PR will be good to go.

**Comment by chris-4444** [comment]
@lijiarui i have updated the post accordingly. kindly approve. thanks.

**Comment by huan** [comment]
ping @lijiarui

File: docusaurus/docs/explanations/overview.mdx [hunk 3]
```diff
@@ -1,7 +1,27 @@
 ---
 slug: /explanations/
-title: Wechaty explanations overview
-sidebar_label: 'explanations: Overview'
+title: Overview of Explanation section
```
Comment (lijiarui) [inline docusaurus/docs/explanations/overview.mdx]: image

It seems the title is too long here, I suggest changing to `Overview` as the other part wrote.

Also, I think we should change `explanation` to `Explanation` as the other part wrote, using the uppercase letter.

**Review by lijiarui — CHANGES_REQUESTED** [review]
This part looks a little bit strange, some of the explanation parts is shown while the other part isn't shown. Could you tell us the reason why you design like this?

**Review by chris-4444 — APPROVED** [review]
Shortened the SDK title as requested.

File: docusaurus/docs/explanations/overview.mdx [hunk 4]
```diff
@@ -1,7 +1,63 @@
...
+
+## [FAQ](faq.md)
+
+TBW
+
+## [DevOps Tools](#placeholder-link)
+
+In this section, we learn about the **DevOps tools and configurations** used by Wechaty while creating chatbots, puppets and certain workflows.
+
+## SDK, RPA, and Conversational Interfaces
```
Comment (lijiarui) [inline docusaurus/docs/explanations/overview.mdx]: change `SDK, RPA, and Conversational Interfaces` to `SDK & CUI`, and we need link for this.

File: docusaurus/docs/explanations/overview.mdx [hunk 5]
```diff
@@ -1,7 +1,63 @@
...
+
+## [Concepts](concepts.md)
+
+TBW
+
+## [Architecture](architecture.md)
+
+The **Architecture** section discusses the fundamental structures of Wechaty chatbot. You will learn how the chatbot works and interacts with other components, how each of the components is represented, and how it all fits as a whole.
+
+## [Life cycle](lifecycle.md)
```
Comment (lijiarui) [inline docusaurus/docs/explanations/overview.mdx]: cycle --> Cycle

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: docusaurus/docs/explanations/overview.mdx (hunk 4 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/overview.mdx]: @lijiarui  The  link can't be added  until #930 has been merged.  Unless you are saying we change it to something like `[SDK & CUI](#placeholder-link)`.

Comment (nibble0101) [reply docusaurus/docs/explanations/overview.mdx]: Changed to `SDK & CUI` and added place holder link.

File: docusaurus/docs/explanations/overview.mdx (hunk 5 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/overview.mdx]: Capitalized cycle.

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Changes requested by Rui have been implemented 👍🏼

**Review by lijiarui — APPROVED** [review]
LGTM

**Pull request merged**